### PR TITLE
Remove the all roles from the department pages

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Removed the all roles tabbed section from the department page as it was failing. This moves the design closer to its destination.

## QA

- Check out the careers overview page
- Click all departments
- Check the department's pages work
- Go to /careers/techops and see that resolves to support engineering
- Submit a test application for your job of choice and check that works

## Issue / Card
Fixes https://github.com/canonical/canonical.com/issues/705

### CODE REVIEW

File: webapp/app.py [hunk 1]
```diff
@@ -285,9 +290,10 @@ def department_group(department_slug):
         if template.endswith(".html"):
             template = template[:-5]
         templates.append(template)
-
+    print(templates, department_slug)
```
Comment (jpmartinspt) [inline webapp/app.py:293]: Sneaky print.

File: templates/careers/support-engineering.html [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
-  <p>In TechOps, you will enjoy the excitement of a team environment that is fun, diverse, distributed globally and incredibly competent. Every day, we work to enable our customers to work on some of the most challenging problems in the world.</p>
+  <p><strong>Support engineering, or Technical Services and Operations role ranges from IT and managed services to developing commercial systems used by our company. We develop, test, ship and operate the systems that drive Canonical and underpin Ubuntu. We believe in a strong customer service ethic and operations through code. The robots we are constructing to run the data centre need to be backed by humans that care about the needs of our users and customers.</strong></p>
+  <p>In TecSupport engineeringhOps, you will enjoy the excitement of a team environment that is fun, diverse, distributed globally and incredibly competent. Every day, we work to enable our customers to work on some of the most challenging problems in the world.</p>
```
Comment (huwshimi) [inline templates/careers/support-engineering.html]: I think this is a copy and paste error.

File: webapp/app.py (hunk 1 above)
Comment (anthonydillon) [reply webapp/app.py]: They gone :)

File: templates/careers/support-engineering.html (hunk 2 above)
Comment (petermakowski) [inline templates/careers/support-engineering.html:11]:
```suggestion
  <p>In Support engineering, you will enjoy the excitement of a team environment that is fun, diverse, distributed globally and incredibly competent. Every day, we work to enable our customers to work on some of the most challenging problems in the world.</p>
```

**Review by huwshimi — APPROVED** [review]
Looks good.

One small QA issue that I picked up (not introduced by this PR) is that on the /careers/departments page the "Check out open Human Resources roles" link lands at "/careers/all?filter=Human+Resources" but none of the filters are ticked. Maybe the link should be to "/careers/all?filter=People"?

**Comment by anthonydillon** [comment]
Thanks, done.

**Pull request merged**